scripts/eloot.lic

```diff
@@ -12,12 +12,14 @@
       contributors: SpiffyJr, Athias, Demandred, Tysong, Deysh, Ondreian
               game: Gemstone
               tags: loot
-           version: 1.5.37
+           version: 1.5.38
```

Comment:
Due to semantic versioning, this is a new feature and so it should go to 1.6.0.

Reply:
Otherwise, works great so far for me.

Reply:
Version updated to 1.6.0